indexing-service/src/main/java/org/apache/druid/indexing/overlord/TaskQueue.java

```diff
@@ -403,7 +403,7 @@ private void startPendingTasksOnRunner()
       updateTaskEntry(taskId, entry -> {
         if (entry == null) {
           unknownTaskIds.add(taskId);
-          shutdownUnknownTaskOnRunner(taskId);
+          shutdownTaskOnRunner(taskId, "Task is not present in queue anymore.");
```

Comment:
Since now the shutdown is running in a new exec service, does it make sense for us to adjust line 412 to mention that we *attempted* to clean up X tasks. 

`
log.info("Attempted to cleaned up [%,d] tasks on task runner with IDs[%s].", unknownTaskIds.size(), unknownTaskIds);
`